### PR TITLE
Add filter_id to message

### PR DESCRIPTION
**Pull request by cshcomcom**

### CODE REVIEW

File: event-handler/event-channel-manager.js [hunk 1]
```diff
@@ -98,9 +98,11 @@ class EventChannelManager {
     };
   }
 
-  transmitEvent(channel, event) {
-    channel.webSocket.send(this.makeMessage(BlockchainEventMessageTypes.EMIT_EVENT,
-        JSON.stringify(event.toObject())));
+  transmitEvent(channel, event, eventFilterId) {
+    const eventObj = event.toObject();
+    eventObj.filter_id = eventFilterId;
```
Comment (platfowner) [inline event-handler/event-channel-manager.js:103]: what about adding this information earlier e.g. in transmitEventByEventFilterId()?
Object.assign() can be used for safety?

Comment (cshcomcom) [reply event-handler/event-channel-manager.js]: Looks better! Done!

Comment (platfowner) [reply event-handler/event-channel-manager.js]: Ah, Object.assign() seems to be optional.

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by cshcomcom** [comment]
Thanks for reviews! Now merging

**Pull request merged**